guides/common/modules/snip_upgrading-your-project-server.adoc

```diff
@@ -109,3 +109,16 @@ endif::[]
 ----
 endif::[]
 include::snip_steps-needs-reboot.adoc[]
+. Perform the following post-upgrade tasks to ensure your environment is adjusted to any changes applied during the upgrade.
```

Comment:
this step is identical to the one in the disconnected guide -- any reason not to use a common snippet for them?